Source/Entities/MOPixel.cpp

```diff
@@ -24,6 +24,7 @@ void MOPixel::Clear() {
 	m_MaxLethalRange = 1;
 	m_LethalSharpness = 1;
 	m_Staininess = 0;
+	m_PostEffectEnabled = true;
```

Comment:
please add a comment describing why this is being done (backwards compat reasons)